test/Sema/parameterized.swift

```diff
@@ -1,165 +0,0 @@
-// SWIFT_ENABLE_TENSORFLOW
```

Comment:
Perhaps we should keep this test (and `Parameterized` protocol definition/synthesis) until the new design is fully baked? That would make me feel more comfortable.

Reply:
It's a failing test and there's no real value to preserve it in the current form. I'll be happy to revert the last change to `Parameterized` so that `Parameterized` is still only connected to `ParameterGroup` and separate from `Differentiable`.